app/tests/unit/components/webade.spec.js

```diff
@@ -0,0 +1,411 @@
+const axios = require('axios');
+const config = require('config');
+const crypto = require('crypto');
+const cryptico = require('cryptico-js');
+const log = require('npmlog');
+const MockAdapter = require('axios-mock-adapter');
+
+const webade = require('../../../src/components/webade');
+const permissionHelpers = require('../../../src/components/permissionHelpers');
+const { acronymService, lifecycleService } = require('../../../src/services');
+const utils = require('../../../src/components/utils');
+
+log.level = config.get('server.logLevel');
+log.addLevel('debug', 1500, { fg: 'cyan' });
```

Comment:
Use the utils.logHelper function in lieu of this.

Reply:
Fixed